assets/node.yaml

```diff
@@ -157,8 +157,6 @@ spec:
             items:
               - key: cloud.conf
                 path: cloud.conf
-              - key: multiaz-cloud.conf
-                path: multiaz-cloud.conf
         - name: cacert
           # Extract ca-bundle.pem to /etc/kubernetes/static-pod-resources/configmaps/cloud-config if present.
```

Comment:
Looks like we deleted the configmap above but these files still refer to non-existant configmap called `openstack-cinder-config` ? We need to update both `controller.yaml` and `node.yaml` to refer to new file?

Reply:
Nice catch :eagle: